mpdev/autogen/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
```

Comment:
We probably should think through the directory structure for these type-specific containers (and refactor them afterward).